lib/class-wp-webfonts-provider-local.php

```diff
@@ -0,0 +1,263 @@
+<?php
+/**
+ * Webfonts API: Provider for locally-hosted fonts.
+ *
+ * @package    WordPress
+ * @subpackage WebFonts
+ * @since      6.0.0
```

Comment:
Hmmm since this API is slated for WP 6.0, it made sense to keep the `@since` comments (like we do in other files as well), so I moved the webfonts-API files to the `compat/wordpress-6.0` folder 👍